crates/rpc/src/rpc.rs

```diff
@@ -96,6 +100,27 @@ impl KoraRpc {
         result
     }
 
+    pub async fn sign_versioned_transaction(
+        &self,
+        request: SignTransactionRequest,
+    ) -> Result<SignTransactionResponse, KoraError> {
+        info!("Sign versioned transaction request: {:?}", request);
+        let result = sign_versioned_transaction(&self.rpc_client, &self.validation, request).await;
+        info!("Sign versioned transaction response: {:?}", result);
+        result
+    }
+
+    pub async fn sign_and_send_versioned_transaction(
+        &self,
+        request: SignAndSendTransactionRequest,
+    ) -> Result<SignAndSendTransactionResponse, KoraError> {
+        info!("Sign and send versioned transaction request: {:?}", request);
+        let result =
+            sign_and_send_versioned_transaction(&self.rpc_client, &self.validation, request).await;
+        info!("Sign and send versioned transaction response: {:?}", result);
+        result
+    }
```

Comment:
what's the reason you need new endpoints for this? can't you just use the same sign_transaction endpoint and parse either versioned or non-versioned transactions?